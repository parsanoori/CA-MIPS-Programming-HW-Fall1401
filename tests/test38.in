5
430592426
541541975
502066595
665071686
336782489