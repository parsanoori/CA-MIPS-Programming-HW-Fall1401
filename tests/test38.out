336782489
430592426
502066595
541541975
665071686